readme.zh_cn.md

```diff
@@ -0,0 +1,44 @@
+<img src="./media/banner.png" alt="无线调试开关小工具"/>
+
+<p align="center">
+    <a href="https://github.com/Smooth-E/wireless-adb-switch/releases/latest" target="_blank" rel="nofollow noopener">
+        <img alt="在 GitHub 上获取" src="media/badge-github.png" width="40%"/>
+    </a>
+    <a href="https://f-droid.org/ru/packages/com.smoothie.wirelessDebuggingSwitch" target="_blank" rel="nofollow noopener">
+        <img alt="在 F-Droid 上获取" src="media/badge-f-droid.png" width="40%"/>
+    </a>
+</p>
+
+> 以其他语言阅读：[`🇷🇺`](./readme.ru.md) [`🇧🇷`](./readme.pt_br.md) [`🇨🇳`](./readme.zh_cn.md) [`翻译成您的语言！`](./translate.md)
+
+
+**WADBS**（**无线安卓调试桥接开关（Wireless Android Debugging Bridge Switch）** 的简称）是一款可以快速开关安卓无线调试功能的应用程序。
+它提供了几个小工具和一个快速设置磁贴来实现这一功能。 
+您还可以使用 KDE Connect 的剪贴板共享功能（参见 [scripts](./scripts)）即时连接到电脑。
+
+## 特性
+
+<img src="./media/feature-qs-tile.png" alt="快速设置磁贴"/>
+<img src="./media/feature-widgets.png" alt="主屏幕小工具"/>
+<img src="./media/feature-customization.png" alt="自定义小工具"/>
+<img src="./media/feature-instant-connection.png" alt="即时连接"/>
+<img src="./media/feature-shizuku.png" alt="与 Shizuku 工作"/>

```

Comment:
The `alt` text here differs from the one you provided in the table and which is displayed on the picture itself. The text here translates as "working with Shizuku". This does not convey the idea that WADBS can also work with root. Please, replace the `alt` text with something like "works with root or Shizuku" or just "root or Shizuku".